evals/funnels/sql/steps.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
+from dataclasses import dataclass
+from typing import List, Dict, Any, Optional, Tuple
+from evals.eval_types import TableDetails, SQLDetails
+from evals.funnels.sql.utils import parse_table_path, run_sql_query
+
+
+@dataclass
+class FunnelStepResult:
+    name: str
+    passed: bool
+    notes: Optional[str] = None
+
+
+def sql_generated_test(
```

Comment:
Maybe we can call this `is_sql_generated_test`, I wasn't sure exactly what this was going to test until I looked at the docstring